modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordLocalServiceImpl.java

```diff
@@ -287,7 +337,7 @@ public void deleteFormInstanceRecords(long ddmFormInstanceId)
 		for (DDMFormInstanceRecord ddmFormInstanceRecord :
 				ddmFormInstanceRecords) {
 
-			deleteFormInstanceRecord(ddmFormInstanceRecord);
+			deleteFormInstanceRecord(ddmFormInstanceRecord, false);
```

Comment:
Since this method is public, I’m concerned about its potential misuse. For instance, a user might attempt to delete all the entries from a form while intending to keep the form itself (i.e., the form remains but without any entries). In such a case, how would the form function without the associated report?

Would it make sense to add a parameter to this method, allowing developers to specify whether the report should be updated or not?